src/lib/viewers/controls/page/PageControls.scss

```diff
@@ -0,0 +1,42 @@
+@import '../styles';
+
+.bp-PageControls {
+    @include bp-ControlGroup;
+}
+
+.bp-PageControls-button {
+    @include bp-ControlButton($width: 32px);
+}
+
+.bp-PageControls-num-button {
+    @include bp-ControlButton($width: auto);
+
+    .bp-PageControls-num-button-label {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        margin: 5px;
+        padding: 7px 5px;
+        color: $white;
+        font-size: 14px;
+        background-color: $fours;
+        border-radius: 3px;
+    }
+}
+
+.bp-PageControls-num-input {
```

Comment:
The SUIT CSS naming convention uses hyphens to indicate hierarchy. We should probably call this `bp-PageControls-numInput` or move the button/input to a separate component entirely (preferable given that it has separate "temporary" state).